lib/foreman_maintain/utils/service/systemd.rb

```diff
@@ -1,17 +1,25 @@
 module ForemanMaintain::Utils
   module Service
     class Systemd < Abstract
-      def initialize(name, priority, _options = {})
+      def initialize(name, priority, options = {})
         super
         @sys = SystemHelpers.new
       end
 
       def command(action, options = {})
+        if %w[enable disable].include?(action) && @name.include?('pulpcore-worker@*')
```

Comment:
I think it'd be better to pass this in as an option, :disable_enablement_control => true  or something like that, and have it be controlled here.  I'd probably handle that in the enable and disable methods rather than in this command method as well.

I'd think that this command module should not have any service-specific knowledge.

Reply:
thats a good point, let me look into that

Reply:
@jlsherrill updated to take an option and use that to skip

Reply:
err... hold on :)

Reply:
@jlsherrill ok, its now updated to only skip on enable and disable with that option :)